google-ads-examples/src/main/java/com/google/ads/googleads/examples/utils/ArgumentNames.java

```diff
@@ -73,6 +73,7 @@
   public static final String FREEFORM_KEYWORD_TEXT = "--freeformKeywordText";
   public static final String GCLID = "--gclid";
   public static final String GEO_TARGET_CONSTANT_ID = "--geoTargetConstantId";
+  public static final String GBRAID = "--gbraid";
```

Comment:
The order seems wrong. Assuming that alphabetic order is used for the variable names, `GBRAID` should be before `GCLID`.

Reply:
Thanks for catching this. Fixed.